platform-cloud/docs/compute-envs/seqera-compute.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: "Seqera Compute"
+description: "Instructions to set up Seqera Compute in Seqera Platform"
+date: "21 Apr 2025"
+tags: [seqera compute, compute environment]
+---
+
+Seqera Compute enables you to run pipelines and studio sessions in Seqera Cloud in a fully managed and optimized AWS environment. Seqera automatically provisions and manages all the underlying resources, including AWS accounts, credentials, roles, AWS Batch compute environments, and S3 storage buckets, requiring minimal user configuration. Using prepaid credits enables control and visibility of the compute spend in each of your organization's workspaces.
+
+### Manage Seqera Compute credits 
+
+Seqera Compute environments consume credits when running pipelines or studio sessions. Credits are consumed for CPU time, memory and storage usage, and network costs. 1 Seqera Compute credit is equivalent to $1 (USD), and resources are charged at the following rates:
+- CPU time: 1 CPU/Hr = 0.1 USD
+- Memory: 1 GiB/Hr = 0.025 USD 
+- Storage: 1 GB = 0.25 USD per month 
+
+:::note 
+Storage and network costs vary per region. Data ingress and egress across regions incure additional cost. 
+:::
+
+View your Seqera Compute credit balance and spend under both the organization and workspace **Settings** tabs. Workspace settings show the credits spent in that workspace, whereas organization settings show the total credits spent in all workspaces in that organization. 
+
+Select **Contact us to upgrade** to request additional credits for your organization. 
+
+### Default limits
+
+Seqera Compute has default workspace limits on compute environments, and organization limits on data storage and CPU cores. 
+
+|                                    | **Basic**                        | **Pro**                            |
+|------------------------------------|----------------------------------|------------------------------------|
+| Cloud storage                      | 25 GB                            | Unlimited                          |
+| Compute environments per workspace | 5                                | 20                                 |
+| Total CPU cores                    | 100                              | 1000                               |
+
+:::info
+[Contact us](https://seqera.io/contact-us/) to discuss custom limits for Pro, academic, or evaluation licenses. 
+:::
+
+### Create a Seqera Compute environment 
+
+1. In a workspace with Seqera Compute enabled, select **Compute environments > New environment**.
+1. Enter a descriptive name for this environment, such as _Seqera Compute 1 (eu-west-1)_.
+1. Under **Platform**, select **Seqera Compute**. 
+1. Select a target execution **Region**. 
+    :::info
+    Seqera Compute is available in the following AWS regions: 
```

Comment:
This list isn't accurate, the list should be – 
United States 
us-east-1 (Northern Virginia, USA)
us-west-2 (Oregon, USA)
us-east-2 (Ohio, USA)
us-west-1 (Northern California, USA)

Europe 
eu-west-1 (Ireland)
eu-west-2 (London, UK)
eu-central-1 (Frankfurt, Germany)
eu-west-3 (Paris, France) 

APAC
ap-southeast-1 (Singapore)

Reply:
I went off the listed available regions in staging: 
<img width="998" alt="image" src="https://github.com/user-attachments/assets/2fdcc7c1-f67e-47b7-9f6b-8f7373f7cb18" />

Is this list trimmed down in prod? I don't have access to a prod workspace where I can create a SC CE

Reply:
Going with restricted list, per https://seqera.slack.com/archives/C081H57DNAX/p1747053981429919